### PR TITLE
add local headless platform and comment out failing tests

### LINKED ISSUE #2: Support headless platform
## Expected Behavior
TestFX should use the headless platform from https://github.com/openjdk/jfx-sandbox/tree/johanvos-headless, and not monocle

## Actual Behavior

To get the fastest possible support, create a [minimal example](https://stackoverflow.com/help/mcve)
and upload it to GitHub.

## Specifications

  - Version:
  - Platform:

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #2

### CODE REVIEW

File: subprojects/testfx-core/testfx-core.gradle [hunk 1]
```diff
@@ -21,12 +21,14 @@ ext.pomName = 'TestFX Core'
 ext.pomDescription = 'TestFX core library'
 ext.moduleName = 'org.testfx'
 ext.openjfxVersion = '23.0.1'
+ext.headlessSDKPath= "${System.properties['user.home']}${File.separator}Downloads${File.separator}javafx-sdk-24"
```
Comment (johanvos) [inline subprojects/testfx-core/testfx-core.gradle:24]: this is a rather strong requirement. Can't we just take an environment variable here? (e.g. System.getEnv("JAVAFX_HOME"))

File: subprojects/testfx-junit5/testfx-junit5.gradle [hunk 2]
```diff
@@ -21,12 +21,14 @@ ext.pomName = 'TestFX JUnit5'
 ext.pomDescription = 'TestFX JUnit5 library'
 ext.moduleName = 'org.testfx.junit5'
 ext.openjfxVersion = '23.0.1'
+ext.headlessSDKPath= "${System.properties['user.home']}${File.separator}Downloads${File.separator}javafx-sdk-24"
```
Comment (johanvos) [inline subprojects/testfx-junit5/testfx-junit5.gradle:24]: same here

File: subprojects/testfx-spock/testfx-spock.gradle [hunk 3]
```diff
@@ -21,6 +21,7 @@ ext.pomName = 'TestFX Spock'
 ext.pomDescription = 'TestFX Spock library'
 ext.moduleName = 'org.testfx.spock'
 ext.openjfxVersion = '23.0.1'
+ext.headlessSDKPath= "${System.properties['user.home']}${File.separator}Downloads${File.separator}javafx-sdk-24"
```
Comment (johanvos) [inline subprojects/testfx-spock/testfx-spock.gradle:24]: same here

File: subprojects/testfx-core/testfx-core.gradle (hunk 1 above)
Comment (jperedadnr) [reply subprojects/testfx-core/testfx-core.gradle]: makes sense, done

**Review by johanvos — APPROVED** [review]

**Pull request merged**